tools/check_examples/README.md

```diff
@@ -11,7 +11,11 @@ and called from `.github/workflows/check_examples.yml`
 
 ## Dependency
 
-The script uses `pdm` and the dependencies are listed in [`pyproject.toml`](pyproject.toml)
+The script uses `markdownlint-cli2` to lint markdown files.
+See [`markdownlint-cli2`](https://github.com/DavidAnson/markdownlint-cli2) for installation instruction.
```

Comment:
no no. Just "npm install". You can also then add "start" script that will run your linter.

Reply:
Done